nextflow.config

```diff
@@ -67,6 +67,13 @@ params {
     build_kraken2            = false
     kraken2_keepintermediate = false
     build_krakenuniq         = false
+
+    // General output options
+    generate_tar_archive = false
+
+    generate_downstream_samplesheets = false
+    generate_pipeline_samplesheets   = null
+    generate_samplesheet_dbtype   = 'raw'
```

Comment:
Does setting generate_samplesheet_dbtype to "tar" automatically set generate_tar_archive to "true"?

Reply:
No, but there should be an input validation warning that blocks it if tar requested in the samplesheet but not set to true